hotel/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Hotel Management Base",
+    "version": "8.0.1.0.0",
+    "author": ["Domatix",
+               "Serpent Consulting Services Pvt. Ltd.",
+               "Odoo Community Association (OCA)"],
```

Comment:
please use a string, with values separated by commas (odoo expects a string not a list)